Add slim images to release process (#23391)
mistercrunch/Airflow: BREEZE.rst

  :width: 100%
  :alt: Breeze verify-prod-image

Releasing Production images to DockerHub
----------------------------------------

The **Production image** can be released by release managers who have permissions to push the image. This

> happens only when there is an RC candidate or final released version of Airflow released.

Reviewer: ```suggestion
happens only when there is an RC candidate or final version of Airflow released.
```

The double released sounds weird to me.

> Typically you can release "regular" and "slim" images separately.

Reviewer: ```suggestion
You release "regular" and "slim" images separately.
```

This implies you can do them at the same time, but you tend to do it one at a time. If I'm not mistaken, you have to do them one at a time, right?

Releasing "regular" images:

.. code-block:: bash

     breeze release-prod-images --airflow-version 2.4.0

> Or "slim" images

Reviewer: ```suggestion
Or "slim" images:
```

.. code-block:: bash

     breeze release-prod-images --airflow-version 2.4.0 --slim-images

By default when you are releasing the "final" image, we also tag image with "latest" tags but this

> step can be skipped if you pas ``--skip-latest`` flag.

Reviewer 2: s/pas/pass/g

> step can be skipped if you pass ``--skip-latest`` flag.

Reviewer: ```suggestion
step can be skipped if you pass the ``--skip-latest`` flag.
```

> Those are all available flags of ``release-prod-images`` command:

Reviewer: ```suggestion
These are of the all available flags for the ``release-prod-images`` command:
```

Reviewer 2: ```suggestion
These are all of the available flags for the ``release-prod-images`` command:
```
The addition `of the` should come after all I reckon.

Reviewer: Oh, yes, thanks 🤦‍♂️
